libs/labelbox/src/labelbox/schema/dataset.py

```diff
@@ -166,47 +166,13 @@ def create_data_row(self, items=None, **kwargs) -> "DataRow":
 
         return self.client.get_data_row(res[0]["id"])
 
-    def create_data_rows_sync(
-        self, items, file_upload_thread_count=FILE_UPLOAD_THREAD_COUNT
-    ) -> None:
-        """Synchronously bulk upload data rows.
-
-        Use this instead of `Dataset.create_data_rows` for smaller batches of data rows that need to be uploaded quickly.
-        Cannot use this for uploads containing more than 1000 data rows.
-        Each data row is also limited to 5 attachments.
-
-        Args:
-            items (iterable of (dict or str)):
-                See the docstring for `Dataset._create_descriptor_file` for more information.
-        Returns:
-            None. If the function doesn't raise an exception then the import was successful.
-
-        Raises:
-            ResourceCreationError: If the `items` parameter does not conform to
-                the specification in Dataset._create_descriptor_file or if the server did not accept the
-                DataRow creation request (unknown reason).
-            InvalidAttributeError: If there are fields in `items` not valid for
-                a DataRow.
-            ValueError: When the upload parameters are invalid
-        """
-        warnings.warn(
-            "This method is deprecated and will be "
-            "removed in a future release. Please use create_data_rows instead."
-        )
-
-        self._create_data_rows_sync(
-            items, file_upload_thread_count=file_upload_thread_count
-        )
-
-        return None  # Return None if no exception is raised
-
     def _create_data_rows_sync(
         self, items, file_upload_thread_count=FILE_UPLOAD_THREAD_COUNT
     ) -> "DataUpsertTask":
         max_data_rows_supported = 1000
         if len(items) > max_data_rows_supported:
             raise ValueError(
-                f"Dataset.create_data_rows_sync() supports a max of {max_data_rows_supported} data rows."
+                f"Dataset._create_data_rows_sync() supports a max of {max_data_rows_supported} data rows."
```

Comment:
We can remove the check for `max_data_rows_supported` and the exception, because all data row creation is async under the hood
Besides, this method is now only used in `create_data_row` that creates just 1 data row